packages/gateway/src/Shard.ts

```diff
@@ -440,6 +450,17 @@ export class DiscordenoShard {
         await this.identify()
         return
       }
+      case GatewayCloseEventCodes.NormalClosure:
```

Comment:
```suggestion
      // NOTE: This case must always be right above the cases that runs with default case because of how switch works when you don't break / return, more info below.
      case GatewayCloseEventCodes.NormalClosure:
```